src/ntt.rs

```diff
@@ -214,4 +196,27 @@ mod tests {
 
         assert_eq!(got, want);
     }
+
+    #[test]
+    fn test_ntt_interpolation() {
+        let count = 128;
+        let points = repeat_with(random).take(count).collect::<Vec<Field128>>();
```

Comment:
nit: clippy always yells at me about doing `std::iter::repeat_n` instead of chaining `repeat_with` and `take`

Reply:
Actually, we have a `random_vector()` for just this